geode-management/src/main/java/org/apache/geode/management/configuration/Pdx.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.configuration;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import org.apache.geode.cache.configuration.CacheElement;
+import org.apache.geode.management.api.CorrespondWith;
+import org.apache.geode.management.api.RestfulEndpoint;
+import org.apache.geode.management.runtime.RuntimeInfo;
+
+public class Pdx extends CacheElement implements RestfulEndpoint, CorrespondWith<RuntimeInfo> {
+  public static final String PDX_ID = "PDX";
+  public static final String PDX_ENDPOINT = "/configurations/pdx";
+
+  protected Boolean readSerialized;
```

Comment:
What is the reason for the Object "Boolean" here instead of the primitive "boolean"? This trickles down to the gettors and settors.

Reply:
the generated code is "Boolean", and there is no default defined in cache.xsd. Do you know what are the defaults of these booleans if they are not specified in the cache.xml?

Reply:
You can see the defaults for these in: org.apache.geode.internal.cache.CacheConfig
I think they just all use the standard java default of "false".
The xml parsing code has some logic that can ask if an attribute was specified in the xsd so it might use a null Boolean to determine if it was unspecified.

Since these are in our external java api I think we would want to go with the primitive "boolean". That way when a user asks what the value is they would always get "true" or "false". With Boolean they could also get "null" which I don't think we want to allow in this context.